package.json

```diff
@@ -25,6 +24,8 @@
     "prop-types": "15.8.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-slick": "0.28.1",
```

Comment:
Good if everything is working for you, though I would be careful in using packages when the following signs are visible on GitHub:
- 80+ Opened PR that are very old
- Appart from the last release (auto update packages for React 18 compatibility, no significant bug fix, while many users reported some and submitted PR for them)
- doc not updated in 2 years
- code not updated in the last 14months

This comment is absolutely not a blocker (especially for an IC), but a reminder to consider it twice in a real project :)

Reply:
Thanks for the note, I'll keep that in mind!